spec/student_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../student'
+
+describe 'testing student class' do
+  it 'should generate both getters and setters' do
+    student = Student.new(18)
+    student.age = 20
```

Comment:
- [ ] Kindly add test to check when initializing the `Student` class (test for constructor method) as you rightly did for book_spec.rb and person_spec.rb files. 👍

- [ ] The test to check if students can `play_hooky` is missing as well. Kindly implement it. 👍